shared/profile/showcase-team-offer/index.tsx

```diff
@@ -98,19 +98,21 @@ const ShowcaseTeamOffer = (props: Props) => {
       {!Styles.isMobile && <ShowcaseTeamOfferHeader />}
       <Kb.ScrollView>
         {Styles.isMobile && <ShowcaseTeamOfferHeader />}
-        {props.teams.map(teamMeta => (
-          <TeamRow
-            key={teamMeta.id}
-            canShowcase={teamMeta.allowPromote || ['admin', 'owner'].includes(teamMeta.role)}
-            isExplicitMember={teamMeta.isMember}
-            name={teamMeta.teamname}
-            isOpen={teamMeta.isOpen}
-            membercount={teamMeta.memberCount}
-            onPromote={promoted => props.onPromote(teamMeta.id, promoted)}
-            showcased={teamMeta.showcasing}
-            waiting={!!props.waiting[teamWaitingKey(teamMeta.teamname)]}
-          />
-        ))}
+        {props.teams
+          .filter(teamMeta => teamMeta.isMember)
+          .map(teamMeta => (
+            <TeamRow
+              key={teamMeta.id}
+              canShowcase={teamMeta.allowPromote || ['admin', 'owner'].includes(teamMeta.role)}
+              isExplicitMember={teamMeta.isMember}
```

Comment:
We should find out why this broke. The rows are supposed to know about this already:
https://github.com/keybase/client/blob/44e32712f1f1f3560efc4462343e60e947be5bdd/shared/profile/showcase-team-offer/index.tsx#L67-L69